src/Client/Web/Playground/Bit.Client.Web.BlazorUI.Playground/Web/Pages/Component.razor

```diff
@@ -363,4 +363,116 @@
     <BitButton OnClick="() => IsToggleChecked = true">Make Toggle Check</BitButton>
     <br />
     <br />
+
+
```

Comment:
Please consider improving these samples visually.
their too simple and give a bad feeling to the viewer :)

Reply:
Added minor changes to samples